pkg/kubelet/cm/event_dispatcher.go

```diff
@@ -41,8 +41,7 @@ type EventDispatcher interface {
 
 	// PostStopPod is invoked after all of a pod's containers have permanently
 	// stopped running, but before the pod sandbox is destroyed.
-	PostStopPod(pod *v1.Pod, cgroupPath string) (*lifecycle.EventReply, error)
-
+	PostStopPod(cgroupPath string) error
```

Comment:
This brings up another point though, which is whether it still makes sense to send the raw cgroup path to the isolator. I tend to think not, since we are proposing to the workgroup that isolators should mostly rely on the Kubelet to write to cgroup (and that we'll expand the isolator response to add support for more resource vectors over time).